plugin/src/main/java/io/github/zrdzn/minecraft/greatlifesteal/user/UserListener.java

```diff
@@ -168,6 +171,16 @@ public void changePlayerHearts(PlayerDeathEvent event) {
                         .map(GreatLifeStealPlugin::formatColor)
                         .forEach(Bukkit::broadcastMessage);
                     break;
+                case BAN:
+                    List<String> reason = action.getParameters().stream()
+                            .map(line -> this.formatPlaceholders(line, victim, killer, lastDamageCause))
+                            .map(GreatLifeStealPlugin::formatColor)
+                            .collect(Collectors.toList());
+                    String formattedReason = String.join("\n", reason);
+                    Bukkit.getBanList(BanList.Type.NAME).addBan(victim.getName(), formattedReason, null, null);
```

Comment:
In #84 I mentioned that it should be based on preventing eliminated player joining, not just banning him.